lib/src/klio/transforms/io.py

```diff
@@ -118,12 +117,11 @@ class KlioReadFromText(beam.io.ReadFromText, _KlioTransformMixin):
     _source_class = _KlioReadFromTextSource
 
 
-class _KlioBigQueryReader(beam_bq_tools.BigQueryReader):
-    def __init__(self, *args, klio_message_columns=None, **kwargs):
-        super(_KlioBigQueryReader, self).__init__(*args, **kwargs)
+class KlioReadFromBigQueryMapper(object):
```

Comment:
Two things:

1. Is there a reason you made this public? I think it could be fine as semi-private. But if public is intentional, can you please add docstrings?
2. If the first answer is yes to public, then docs will have to be updated [here](https://github.com/spotify/klio/blob/master/docs/src/reference/lib/api/transforms/index.rst#docio) and [here](https://github.com/spotify/klio/blob/master/docs/src/reference/lib/api/transforms/io.rst).

Reply:
ah yeah it doesn't need to be public, I'll fix that before merging.